src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda

```diff
@@ -47,3 +48,16 @@ module _ {p} {P : Pred A p} where
     cong here ∘′ subst-injective x≡y ∘′ here-injective
   lookup-injective {p = _ ∷ _}    {there v} {there w} =
     cong there ∘′ lookup-injective ∘′ there-injective
+
+------------------------------------------------------------------------
+-- Relationships to other predicates
+
+open Data.List.Relation.Binary.Sublist.Propositional renaming (lookup to Any-resp-⊆)
```

Comment:
This needs to be made public. But actually I think a better solution is to give the full type of `Any-resp-⊆` and then simply define it in terms of `lookup`.